scripts/test.ts

```diff
@@ -92,9 +92,10 @@ export default function(args: ParsedArgs, logger: logging.Logger) {
   if (args['ve']) {
     // tslint:disable-next-line:no-console
     console.warn('********* VE Enabled ***********');
-  } else if (args.shard !== undefined) {
+  } else {
     // CI is really flaky with NGCC
     // This is a working around test order and isolation issues.
+    console.warn('********* Running ngcc ***********');
     execSync('./node_modules/.bin/ngcc', { stdio: 'inherit' });
```

Comment:
We should probably change this to `./node_modules/.bin/ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points`


So that when running the tests locally you can easily switch between VE and Ivy runs as otherwise you'd need to remove all `node_modules`.

Reply:
Yes, that would be better.